doc/user_manual/metrics.tex

```diff
@@ -411,7 +411,7 @@ \subsection{Dynamic Time Warping}
 \begin{itemize}
   \item \xmlNode{order},          \xmlDesc{int, required field},    order of the DTW calculation: $0$ specifices a classical DTW caluclation and $1$ specifies
```

Comment:
I honestly do not get why we need integer here to specify a enumeration list.
Can we change this in ```classical``` and ```derivative```.
@mandd . In addition, can we add a bit more info regarding what this option is supposed to trigger?????